### PR TITLE
feat(TDI-44284):Rename old tEXAxxx connectors in tExasolxxx

### PR DESCRIPTION
**Pull request by pyzhou-talend**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-44284
**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tExasolCommit/tExasolCommit_java.xml [hunk 1]
```diff
@@ -1,53 +1,53 @@
...
+    <CONNECTOR CTYPE="COMPONENT_OK"/>
+    <CONNECTOR CTYPE="COMPONENT_ERROR"/>
+    <CONNECTOR CTYPE="RUN_IF"/>
+  </CONNECTORS>
+
+  <PARAMETERS>
+    <PARAMETER
+      NAME="CONNECTION"
+      FIELD="COMPONENT_LIST"
+      FILTER="tDBConnection"
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tExasolCommit/tExasolCommit_java.xml:38]: should be **tExasolConnection**, no ?

File: main/plugins/org.talend.designer.components.localprovider/components/tExasolClose/tExasolClose_messages.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
-HELP=org.talend.help.tEXAClose
+HELP=org.talend.help.tExasolClose
 LONG_NAME=Close a selected EXASolution connection.
 CONNECTION.NAME=Component List
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tExasolClose/tExasolClose_messages.properties:3]: i think we only keep the default properties. and create babili jira to let them rename other language on babili system.

File: main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/RenameEXAMigrationTask.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
+// ============================================================================
```
Comment (jzhao-talend) [inline main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/RenameEXAMigrationTask.java:1]: seems missing change of tCreateTable

File: main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/RenameEXAMigrationTask.java [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    public ExecutionResult execute(Item item) {
+        ProcessType processType = getProcessType(item);
+        if (processType == null) {
+            return ExecutionResult.NOTHING_TO_DO;
+        }
+        try {
+            for (int i = 0; i < source.length; i++) {
+                final int j = i;
+                IComponentFilter filter = new NameComponentFilter(source[i]);
+                RenameComponentConversion renameConversion = new RenameComponentConversion(target[i]);
```
Comment (jzhao-talend) [inline main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/RenameEXAMigrationTask.java:54]: when backport to master, please pay attention https://jira.talendforge.org/browse/TUP-26439
which tup team didn't backport to 7.3

File: main/plugins/org.talend.designer.components.localprovider/components/tExasolCommit/tExasolCommit_java.xml (hunk 1 above)
Comment (pyzhou-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tExasolCommit/tExasolCommit_java.xml:38]: Yes. You are right, will fix it.

File: main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/RenameEXAMigrationTask.java (hunk 4 above)
Comment (pyzhou-talend) [reply main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/RenameEXAMigrationTask.java:54]: I will pay attention, thanks

**Pull request merged**